### PR TITLE
Add a MAINTAINERS to list all members of all roles

### PR DESCRIPTION
**Pull request by ahrtr**
As etcd maintainers discussed, we add a separate MAINTAINERS file for each project under etcd-io organization. 

Currently there are only three members in the existing `maintainers-bbolt` github team, so I just added the same list into MAINTAINERS in bbolt. Please anyone feel free to let me know if you have any concerns or comments. 

cc @mitake @ptabor @serathius @spzala @tbg

### CODE REVIEW

File: MAINTAINERS
Comment (serathius) [inline MAINTAINERS:18]: Don't think members need a special section. As mentioned in https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md#member members are defined by being added to an etcd.io org.

Comment (ahrtr) [reply MAINTAINERS]: thx & removed.

**Review by ptabor — APPROVED** [review]

**Comment by ahrtr** [comment]
> but I feel probably a good idea to keep all etcd maintainers here as well (and other repos under etcd-io) to start with.

Thanks @spzala for the feedback. I am OK with this, reasons:
1. I think a mature community also needs maintainer with both technical skill and excellent soft skill. The existing etcd maintainers already demonstrate such skills.
2. The list isn't static; instead the maintainer can retire voluntarily or per [#retiring](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md#retiring)

WDYT?  @mitake @ptabor @serathius @spzala

**Comment by ptabor** [comment]
> > but I feel probably a good idea to keep all etcd maintainers here as well (and other repos under etcd-io) to start with.
> 
> Thanks @spzala for the feedback. I am OK with this, reasons:
> 
> 1. I think a mature community also needs maintainer with both technical skill and excellent soft skill. The existing etcd maintainers already demonstrate such skills.
> 2. The list isn't static; instead the maintainer can retire voluntarily or per [#retiring](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md#retiring)
> 
> WDYT? @mitake @ptabor @serathius @spzala

There are 2 goals: 
 1. Know the list of formal maintainers with the 'voting' privileges.
 2. Know whom to seek for technical expertise and judgement for the given base-code. 

Currently we are putting this as a single 'Maintainers' file, while 2. logically is closer to the OWNERS file. 

To not create too many abstractions, how about keeping both in a single file, but annotate a few maintainers with: 
`#owner` / `#domain-expert`  comment.

**Comment by mitake** [comment]
@ahrtr Thanks for opening the discussion. I think it's a good idea and ok to be a maintainer without the tags. I'll try to find time for catching up the latest status of the project.

**Comment by ahrtr** [comment]
ping @ptabor @serathius . 

I assume this PR is already approved by @mitake and @spzala .  I updated this PR after @ptabor approved, so PTAL. 

I expect we can count on lgtm from all maintainers for:
1. test case changes;
2. CMD changes (not including surgery commands);
3. typo or comment changes;

For another other changes, we should get approval from @ptabor and/or @ahrtr .

**Review by ptabor — APPROVED** [review]

**Pull request merged**
